tensorflow/core/util/gpu_kernel_helper.h

```diff
@@ -53,9 +53,21 @@ using gpuStream_t = hipStream_t;
 using gpuError_t = hipError_t;
 #endif
 
-#define GetGPUStream(context) context->eigen_gpu_device().stream()
 
 namespace tensorflow {
+
+inline const gpuStream_t& GetGpuStream(OpKernelContext* context) {
```

Comment:
Please keep the original GetCudaStream and forward to here. I'm pretty sure we are still referencing it internally.

Reply:
I added back the `GetCudaStream()` in gpu_launch_config.h, and added a comment above indicating that this `GetGpuStream()` is an exact copy of it. 

In case you wondered why I didn't use the `CREATE_CUDA_HOST_FUNCTION_ALIAS` infrastructure. It is because `GetCudaStream()` includes cuda only data structures aliased in this file, i.e. `cudaStream_t`. I have to make a text copy in order to make those two have the right data structure.